src/extensions/sonarcloud/overview.md

```diff
@@ -8,14 +8,14 @@ This Azure DevOps extension provides build tasks that you can add in your build
 
 To get started in a few minutes, you can:
 
-- Follow this dedicated [Microsoft Lab][getstarted]
```

Comment:
[Azure Labs will sunset in 2027](https://learn.microsoft.com/en-us/answers/questions/1818522/azure-lab-services-will-be-retired-on-june-28-2027) and the existing webpage is already down. The replacement points to the .md source file for that tutorial.